docs/components/combobox/index.md

```diff
@@ -0,0 +1,18 @@
+<script setup>
+  import React from './react.md';
+</script>
+
+# Combobox (also known as an autocomplete or autosuggest)
+
+A combobox is the combination of an `<input type="text"/>` and a list. The list is designed to help the user arrive at a value, but the value does not necessarily have to come from that list. Don't think of it like a `<select/>`, but more of an input with some suggestions. You can, however, validate that the value comes from the list, that's up to your app.
+
+## Example
+
+![Combobox](/combobox.png)
+
+<tabs-content>
+  <template #react>
+   <react />
+  </template>
+</tabs-content>
+image.png
```

Comment:
looks like this has accidentally slipped in:)

Reply:
Nicely spotted. Removed now